etc/org.opencastproject.transcription.ibmwatson.IBMWatsonTranscriptionService.cfg

```diff
@@ -37,3 +37,6 @@ ibm.watson.password=<SERVICE_PSW>
 # Email to send notifications of errors. If not entered, the value from
 # org.opencastproject.admin.email in custom.properties will be used.
 #notification.email=
+
+# Start transcription job load
+job.load.start.transcription=0.1
```

Comment:
Since this is set to the default, can we state the default and comment out the setting. That way it's clear that the default value is used. A lot of other configuration keys use this format already. It helps quickly identify values a user has modified:

```
# Start transcription job load
# Default: 0.1
￼#job.load.start.transcription=0.1
```

Reply:
Done! Also for MH-12893.